modelscope/trainers/trainer.py

```diff
@@ -9,6 +9,7 @@
 
 import json
 import torch
+from swift import PeftConfig, Swift, SwiftConfig
```

Comment:
move to line 274, inside the if condition, and add swift import chck util in import_util.py,  make swift an optional dependency of modelscope